tests/test_constant.py

```diff
@@ -0,0 +1,11 @@
+from egcg_core import constants as cst
+
+
+def test_constants_uniqueness():
+    constants = [item for item in dir(cst) if not item.startswith("__")]
+    uniq_constants = set()
+    for constant in constants:
+        if constant in uniq_constants:
+            assert False
+        uniq_constants.add(constant)
```

Comment:
- The `assert True` below will have no effect during a test
- The `assert False` above could just be `assert constant not in uniq_constants`